data/assets/scene/solarsystem/heliosphere/todayssun/grid.asset

```diff
@@ -0,0 +1,119 @@
+local sunTransforms = asset.require("scene/solarsystem/sun/transforms")
+
+
+
+local size = 700000000
+
+local CarringtonPrimeMeridian = {
+  Identifier = "CarringtonPrimeMeridian",
+  Parent = sunTransforms.SunIAU.Identifier,
+  Transform = {
+    Translation = {
+      Type = "StaticTranslation",
+      Position = { 3500000, 0, 0 }
+    },
+    Rotation = {
+      Type = "StaticRotation",
+      Rotation = { 0.0, 0.0, -math.pi / 2.0 } -- -90 degrees
+    },
+    Scale = {
+      Type = "StaticScale",
+      Scale = 698000000
+    }
+  },
+  Renderable = {
+    Type = "RenderableSphericalGrid",
+    Size = size,
+    Color = { 1.0, 1.0, 0.0 },
+    LineWidth = 2.0,
+    LongSegments = 2,
+    LatSegments = 64
+  },
+  GUI = {
+    Name = "Carrington Prime Meridian",
+    Path =  "/Solar System/Heliosphere",
+    Description = [[An arch showing The Carrington prime meridian of the Sun.]] ..
+      [[The line from pole to pole at 0 degree longitude.]]
+  }
+}
+
+local WSA_GridSlice = {
+  Identifier = "WSA_GridSlice",
+  Parent = transforms.HeliocentricEarthEquatorial.Identifier,
+  Transform = {
+    Rotation = {
+      Type = "StaticRotation",
+      Rotation = { 0.0, 0.0, -math.pi / 2.0 } -- -90 degrees
+    },
+    Scale = {
+      Type = "StaticScale",
+      Scale = size
+    }
+  },
+  Renderable = {
+    Type = "RenderableSphericalGrid",
+    Size = size,
+    Color = { 0.8, 0.8, 0.8 },
+    LineWidth = 2.0,
+    LongSegments = 2,
+    LatSegments = 64
+  },
+  GUI = {
+    Name = "Solar Longitude Facing the Earth",
+    Path =  "/Solar System/Heliosphere",
+    Description = [[An arch on the Sun suface that always faces Earth.]]
+  }
+}
+
+local WSA_Grid10Degrees = {
+  Identifier = "WSA_Grid10Degrees",
+  Parent = transforms.HeliocentricEarthEquatorial.Identifier,
+  Transform = {
+    Scale = {
+      Type = "StaticScale",
+      Scale = size
+    }
+  },
+  Renderable = {
+    Type = "RenderableSphericalGrid",
+    Size = size,
+    Color = { 0.035, 0.675, 0.255 },
+    LineWidth = 1.0,
+    Segments = 36
+  },
+  GUI = {
+    Name = "Grid on Sun",
+    Path =  "/Solar System/Heliosphere",
+    Description = [[A grid aligned with the Sun-Earth line, with a 10-degree ]] ..
+      [[seperation between line segments by default.]]
+  }
+}
+
+asset.onInitialize(function()
+  openspace.addSceneGraphNode(CarringtonPrimeMeridian)
+  openspace.addSceneGraphNode(WSA_GridSlice)
+  openspace.addSceneGraphNode(WSA_Grid10Degrees)
+end)
+
+asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(WSA_Grid10Degrees)
+  openspace.removeSceneGraphNode(WSA_GridSlice)
+  openspace.removeSceneGraphNode(CarringtonPrimeMeridian)
+end)
+
+asset.export(CarringtonPrimeMeridian)
+asset.export(WSA_GridSlice)
+asset.export(WSA_Grid10Degrees)
+
+
+
+asset.meta = {
+  Name = "Real-time Sun Grid Lines",
+  Version = "1.0",
+  Description = [[Grids to guide the orientation of the Sun and
+    its rotation in relation to Earth.]],
```

Comment:
Finding this sentence a bit hard to understand (sounds like the grids help the sun, rather that the user :) ), and it would be good if it also included information about what is in the asset. Maybe something like: 

```suggestion
  Description = [[Grids that help show the Sun's orientation and rotation in relation to 
    Earth, including the Carrington prime meridian and Earth-facing longitudes.]],
```
?